devel-git-deps.yaml

```diff
@@ -2,7 +2,7 @@ repositories:
   aligator:
     type: git
     url: git@github.com:edantec/aligator.git
-    version: devel
+    version: temporary_fix
```

Comment:
Just to double-check, it is required to be on the temporary fix branch and not devel or main ?

Reply:
Yes, it is required since the cycling of the parallel solver of Aligator is still broken. Hence, you have to use the temporary fix I introduced in this branch.